src/server/utils/loadModules.js

```diff
@@ -25,9 +26,17 @@ import { setClientModuleMapCache } from './clientModuleMapCache';
 import { updateCSP } from '../middleware/csp';
 import addBaseUrlToModuleMap from './addBaseUrlToModuleMap';
 
+let cashedModuleMapHash;
+
 const loadModules = async () => {
   const moduleMapResponse = await fetch(process.env.HOLOCRON_MODULE_MAP_URL);
   const moduleMap = addBaseUrlToModuleMap(await moduleMapResponse.json());
+
+  const moduleMapHash = hash(moduleMap);
+  if (cashedModuleMapHash && cashedModuleMapHash === moduleMapHash) {
+    return {};
```

Comment:
Is there value in logging here, maybe in the non-cache-hit case? so those who watch logs can see '3 modules change since last poll' and have confirmation that the change has actually been processed?

Reply:
We already log that

https://github.com/americanexpress/one-app/blob/531958f5d20b30900efd34db88fae7a6c5e8406b/src/server/utils/pollModuleMap.js#L151